lib/ex_fizz_buzz_enterprise/input_worker.ex

```diff
@@ -22,13 +21,24 @@ defmodule ExFizzBuzzEnterprise.InputWorker do
     GenServer.cast(__MODULE__, :next)
```

Comment:
Actually, after looking at the stuff below, `start_sequence` or something might be better?